### PR TITLE
Contact staged-recipes for merging

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ Currently available teams are:
 If your PR doesn't fall into those categories please contact
 the full review team `@conda-forge/staged-recipes`.
 
+Once the PR is ready for review, please contact
+`@conda-forge/staged-recipes` to have it merged.
+
```
Comment (beckermr) [inline .github/pull_request_template.md:24]: Can we put in the admin command here?

I think it is `conda-forge-admin please ping staged-recipes` or something like that?

**Comment by carterbox** [comment]
If our policy is that recipe PRs should ping `staged-recipes` for merging, then no one will use the language tags (that we just automated) because they will only ping `staged-recipes`?

And as a reviewer, I want users to use the language specific labels because I don't want to review recipes that are outside my language experience.

**Comment by BastianZim** [comment]
@beckermr This was your suggestion, can you comment on it?

**Comment by BastianZim** [comment]
But I think the gist is that the teams aren't really organised right now with some being empty and sometimes the best reviewer not being part of that team. So more PRs might fall through the cracks than being correctly reviewed.

**Comment by beckermr** [comment]
I really appreciate your comments and am happy to defer to whatever you'd like.

**Pull request merged**